### PR TITLE
Lg/tutorial sandbox 1

### PR DESCRIPTION
**Pull request by lisa-parity**
First tutorial, simplified for the new structure.

### CODE REVIEW

File: v3/how-to-guides/01-basics/a-pallet-integration/index.mdx [hunk 1]
```diff
@@ -15,89 +15,89 @@ You will learn how to include a custom internal pallet that implements an event
...
-/* --snip-- */
-    Something: pallet_something::{Pallet, Call, Storage, Event<T>},
-/* --snip-- */
-  }
-);
-```
-
-> **NOTE:** To add an external pallet, you use a similar method to the one you used with the local pallet, but you must include all of the types your pallet exposes.
-> You must also include the relevant parameter types and constants. For examples of how to declare parameters and constants, see [`pallet_timestamp`][timestamp-frame].
+  ```rust
```
Comment (NukeManDan) [inline v3/how-to-guides/01-basics/a-pallet-integration/index.mdx:39]: sorry this fix was reverted :( I broke the site in #534 - I will patch this on `main` very shortly to get all the fixes in automatically - you don't need to hand do this anymore. Sorry about the trouble!

**Review by NukeManDan — APPROVED** [review]

File: v3/how-to-guides/01-basics/a-pallet-integration/index.mdx (hunk 1 above)
Comment (NukeManDan) [reply v3/how-to-guides/01-basics/a-pallet-integration/index.mdx:39]: Will be included here: https://github.com/substrate-developer-hub/substrate-docs/pull/551
you can _rebase_ on `main`:
```
git checkout main
git pull
git checkout LG/tutorial-sandbox-1
git rebase main
```
And then use an IDE to easily resolve any conflicts.

File: v3/how-to-guides/02-pallet-design/a-add-contracts-pallet/index.mdx [hunk 2]
```diff
@@ -32,80 +32,80 @@ Implement the Contracts pallet [configuration traits][contracts-config-rustdocs]
...
 
-Notice how the above parameter type depends on `WeightInfo`. This requires you to add the following to the top of `runtime/src/lib.rs`:
+  Notice how the above parameter type depends on `WeightInfo`. This requires you to add the following to the top of `runtime/src/lib.rs`:
 
-```rust
-use pallet_contracts::weights::WeightInfo;
-````
+  ```rust
+  use pallet_contracts::weights::WeightInfo;
+  ````
```
Comment (sacha-l) [inline v3/how-to-guides/02-pallet-design/a-add-contracts-pallet/index.mdx:79]:
```suggestion
  ```
```

File: v3/how-to-guides/01-basics/a-pallet-integration/index.mdx (hunk 1 above)
Comment (sacha-l) [reply v3/how-to-guides/01-basics/a-pallet-integration/index.mdx:39]: @NukeManDan : You mean this branch isn't from latest and consequently hasn't been updated with recent fixes?

**Pull request merged**